packages/dev/core/src/Maths/math.vector.ts

```diff
@@ -2515,6 +2515,21 @@ export class Vector3 {
         return result;
     }
 
+    /**
+     * Reflects a vector off the plane defined by a normal
+     * @param inDirection defines the vector direction
+     * @param normal defines the normal

```

Comment:
You should indicate that the normal must be normalized for the result to be correct.

Reply:
Ok, but should I still normalize in the code?

Reply:
Let s not normalize in the code and just add it in the code ?

Reply:
You mean add it in the doc?

Reply:
yup

Reply:
Ok, removing the normalize in the actual code and updating the doc